app/models/welcome_notification.rb

```diff
@@ -1,2 +1,4 @@
 class WelcomeNotification < ApplicationRecord
+  has_one :broadcast, as: :broadcastable
```

Comment:
Same question here re: validating that a `WelcomeNotification` always has an associated `Broadcast`